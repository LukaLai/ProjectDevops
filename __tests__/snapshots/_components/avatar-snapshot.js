import { render } from '@testing-library/react'
import Avatar from '../../../src/app/_components/avatar'

it('renders homepage unchanged', () => {
    const { container } = render(<Avatar name={'jj'} picture={'/public/assets/blog/authors/jj.jpeg'} />)
    expect(container).toMatchSnapshot()
})