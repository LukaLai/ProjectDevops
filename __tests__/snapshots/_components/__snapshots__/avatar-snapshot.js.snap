// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="flex items-center"
  >
    <img
      alt="jj"
      class="w-12 h-12 rounded-full mr-4"
      data-nimg="1"
      decoding="async"
      height="12"
      loading="lazy"
      src="/_next/image?url=%2Fpublic%2Fassets%2Fblog%2Fauthors%2Fjj.jpeg&w=32&q=75"
      srcset="/_next/image?url=%2Fpublic%2Fassets%2Fblog%2Fauthors%2Fjj.jpeg&w=16&q=75 1x, /_next/image?url=%2Fpublic%2Fassets%2Fblog%2Fauthors%2Fjj.jpeg&w=32&q=75 2x"
      style="color: transparent;"
      width="12"
    />
    <div
      class="text-xl font-bold"
    >
      jj
    </div>
  </div>
</div>
`;
